### ISSUE TITLE
Option to collapse the debug panel on load

### ISSUE THREAD

**Issue by MathieuLoutre** [boardgameio/boardgame.io#1039]
Is there a way to hide the debug panel by default and if not, would you take a PR for an option to do that? Additionally, would you accept a PR for an option to hide the new button for the debug panel?

My use case is that I bring up the panel to switch between players when testing but I don't need it shown by default. The new button also interferes with parts of my UI.

**Comment by delucis** [comment]
Happy to see a PR for an option to have the panel collapsed on load. And if it’s helpful, I’d also merge a PR to opt-in to hiding the new show/hide button. Or we could even do this via CSS custom properties to allow control over the button position.

```css
.debug-panel {
  --bgio-debug-collapser-display: none;
  --bgio-debug-collapser-top: auto;
  --bgio-debug-collapser-bottom: .5em;
}
```

**Comment by MathieuLoutre** [comment]
Thanks @delucis! I looked at the possibility of adding control via CSS Variables (which is a nice idea) but it seemed more convoluted. That said, the approach I've taken doesn't seem incompatible with adding further options for positioning down the line.

**Comment by delucis** [comment]
Implemented in #1040, released in https://github.com/boardgameio/boardgame.io/tree/v0.49.8 — thanks @MathieuLoutre  🎉
